bin/docker-test-proxy

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -eu
+
+if [ $# -ne 0 ]; then
+    echo "no arguments allowed for $(basename $0), given: $@" >&2
+    exit 64
+fi
+
+bindir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+rootdir="$( cd $bindir/.. && pwd )"
+
+docker build -f $rootdir/proxy/Dockerfile . \
+    --target build \
+    -t proxy-build \
+    --build-arg=PROXY_UNOPTIMIZED=1 &&
```

Comment:
I'd just remove the ` && ` since it's redundant

you may or may not want to use `docker_build proxy ... --target=build --build-arg=...`, since this will make it interact well with `DOCKER_TRACE`, rtc

Reply:
I didn't use `docker_build` because it looks like it sets `-t` and I wasn't sure how that could be overridden. Also, I figured we'd probably always want to run tests in debug mode?